src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
```

Comment:
Consider adding smooth transition effect on hover to change card title color

Reply:
Thank you, all changes done. 
:)